### PR TITLE
gamedev into main

### PR DESCRIPTION
**Pull request by olelenz**

### CODE REVIEW

File: app/src/main/java/eco/gecko/ecogecko/Board.kt
Comment (daschdy) [inline app/src/main/java/eco/gecko/ecogecko/Board.kt:1]: ok

**Review by daschdy — APPROVED** [review]

File: app/src/main/java/eco/gecko/ecogecko/Board.kt [hunk 1]
```diff
@@ -1,59 +0,0 @@
...
-        @JvmStatic
-        fun newInstance(param1: String, param2: String) =
-            Board().apply {
-                arguments = Bundle().apply {
-                    putString(ARG_PARAM1, param1)
-                    putString(ARG_PARAM2, param2)
-                }
-            }
-    }
-}
```
Comment (daschdy) [inline app/src/main/java/eco/gecko/ecogecko/Board.kt:59]: back in

File: app/src/main/java/eco/gecko/ecogecko/FirstFragment.kt [hunk 2]
```diff
@@ -1,44 +0,0 @@
...
-        binding.buttonFirst.setOnClickListener {
-            findNavController().navigate(R.id.action_FirstFragment_to_SecondFragment)
-        }
-    }
-
-    override fun onDestroyView() {
-        super.onDestroyView()
-        _binding = null
-    }
-}
```
Comment (daschdy) [inline app/src/main/java/eco/gecko/ecogecko/FirstFragment.kt:44]: back in

File: app/src/main/java/eco/gecko/ecogecko/GameBoard.kt [hunk 3]
```diff
@@ -1,5 +1,136 @@
 package eco.gecko.ecogecko
 
-class GameBoard {
```
Comment (daschdy) [inline app/src/main/java/eco/gecko/ecogecko/GameBoard.kt:3]: loook

File: app/src/main/java/eco/gecko/ecogecko/Tile.kt [hunk 4]
```diff
@@ -1,5 +1,220 @@
 package eco.gecko.ecogecko
 
-class Tile {
```
Comment (daschdy) [inline app/src/main/java/eco/gecko/ecogecko/Tile.kt:3]: loook

**Review by daschdy — CHANGES_REQUESTED** [review]

**Review by daschdy — APPROVED** [review]

**Comment by olelenz** [comment]
Merged

**Pull request closed without merging**